packages/erc3kjs/src/lib/abi/ERC3000Registry.json

```diff
@@ -1 +1,88 @@
-[{"anonymous":false,"inputs":[{"indexed":true,"internalType":"contract ERC3000Executor","name":"dao","type":"address"},{"indexed":false,"internalType":"contract ERC3000","name":"queue","type":"address"},{"indexed":true,"internalType":"address","name":"registrant","type":"address"},{"indexed":false,"internalType":"string","name":"name","type":"string"}],"name":"Registered","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"internalType":"contract ERC3000Executor","name":"dao","type":"address"},{"indexed":false,"internalType":"bytes","name":"metadata","type":"bytes"}],"name":"SetMetadata","type":"event"},{"inputs":[{"internalType":"string","name":"","type":"string"}],"name":"nameUsed","outputs":[{"internalType":"bool","name":"","type":"bool"}],"stateMutability":"view","type":"function"},{"inputs":[{"internalType":"contract ERC3000Executor","name":"_dao","type":"address"},{"internalType":"contract ERC3000","name":"_queue","type":"address"},{"internalType":"string","name":"_name","type":"string"},{"internalType":"bytes","name":"_initialMetadata","type":"bytes"}],"name":"register","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"bytes","name":"_metadata","type":"bytes"}],"name":"setMetadata","outputs":[],"stateMutability":"nonpayable","type":"function"}]
\ No newline at end of file
+[
```

Comment:
Let's delete this file, and just reference the artifacts in the `govern-core` repo directly.